Assets/StreamingAssets/Data/Furniture.xml

```diff
@@ -193,6 +192,9 @@
 
     <LocalizationCode>furn_mining_drone_station</LocalizationCode>
     <UnlocalizedDescription>furn_mining_drone_station_desc</UnlocalizedDescription>
+    <RequiredAtmosphere>
+      <Gas hasValue="true" name="O2" />
```

Comment:
The order of the attributes makes no difference what-so-ever.  Is there a particular reason why you want it this way?  I don't feel like that is a worthwhile change to make.

I can certainly add that in... I actually had that originally.

You don't need to supply RequiredAtmosphere, it is completely optional.  In fact, there are some examples of that in the file already.  The "hasValue" attribute is meant to be used for when you SHOULDN'T have that type of atmosphere.  If it doesn't matter at all, then you leave it off.